notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb

```diff
@@ -1514,7 +1295,7 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "gRkvzr2-LgA9"
+        "id": "60092cfdd571"
       },
       "source": [
         "# Batch Prediction Job on the Model"
```

Comment:
- change: Batch Prediction Job on the Model
    - to: Batch prediction job on the model